lib/active_model_serializers/json_pointer.rb

```diff
@@ -4,7 +4,7 @@ module JsonPointer
 
     POINTERS = {
       attribute:    '/data/attributes/%s'.freeze,
-      primary_data: '/data'.freeze
+      primary_data: '/data%s'.freeze
```

Comment:
Would you mind linking to an open issue or creating one on https://github.com/jruby/jruby for the greater good? :)